ports/unix/main.c

```diff
@@ -642,6 +642,14 @@ MP_NOINLINE int main_(int argc, char **argv) {
         }
     }
 
+    #if MICROPY_PY_SYS_UATEXIT
```

Comment:
The whole `sys.uatexit()` functionality needs to be in a completely separate PR, it's not needed for `sys.settrace()` (as far as I can tell).